src/uharfbuzz/_harfbuzz.pyx

```diff
@@ -420,6 +420,26 @@ cdef class Font:
         packed = name
         return packed.decode()
 
+    def draw_glyph_to_pen(self, gid: int, pen):
```

Comment:
Usually one draws a glyph _with_ a pen, not _to_. I would have called this method `draw_glyph_with_pen`

Also, do we need to create a new DrawFuncs() every time one calls draw_glyph_with_pen? Can't we create one for each Font instance and reuse?

Reply:
> Usually one draws a glyph with a pen, not to. I would have called this method draw_glyph_with_pen

True. The whole "pen" thing in my mind has become so abstract that I don't think about it that way anymore. (Just like I don't associate the computer concept of "file" with a physical file to keep papers in.)

That said, I won't mind if this were changed. We should decide before the next release, though, as this PR has already been merged. (Speaking of which: I don't know how to update the harfbuzz submodule to the latest release.)

> Also, do we need to create a new DrawFuncs() every time one calls draw_glyph_with_pen? Can't we create one for each Font instance and reuse?

That is a good point, but could be implemented later as an optimization.

Reply:
> I don't know how to update the harfbuzz submodule to the latest release

you `cd` into the submodule directory, git pull and then checkout the tagged commit, then cd .. back to the root of the parent repo and git add and git commit the change

Reply:
@simoncozens, would you like to revisit the naming of this method, or is this good to go?

Reply:
I’m happy with it myself, but we should probably rename it for other people’s sanity. :-) will send a PR in the morning.

Reply:
> draw_glyph_with_pen

FWIW what I expected was just `draw_glyph` that would then probe pen to see what protocol to talk to it; ie. duck-typing method-overloading.